x/oracle/keeper/aggregator/aggregator.go

```diff
@@ -48,6 +48,34 @@ type aggregator struct {
 	dsPrices map[uint64]string
 }
 
+func (agg *aggregator) copy4CheckTx() *aggregator {
+	ret := &aggregator{
+		finalPrice:  big.NewInt(agg.finalPrice.Int64()),
+		reportPower: big.NewInt(agg.reportPower.Int64()),
+		totalPower:  big.NewInt(agg.totalPower.Int64()),
+
+		reports:  make([]*reportPrice, 0, len(agg.reports)),
+		dsPrices: agg.dsPrices,
+	}
+	for k, v := range agg.dsPrices {
```

Comment:
there are dsXXX  detXXX  for Deterministic. this abbreviation naming is not easy to maintain and not clear for readers to understanding.
It is recommended uniform the **dsPrices** with **detPrices** (deterministic) and add clear comment for the  related variable.

Reply:
ds is for deterministic-source, corresponding to ns: nondeteministic-source
detID for deterministicID